rails_event_store_active_record/lib/rails_event_store_active_record/generators/templates/limit_for_event_id_template.rb

```diff
@@ -0,0 +1,6 @@
+class LimitForEventId < ActiveRecord::Migration<%= migration_version %>
```

Comment:
in specs:
```
     NameError:
       uninitialized constant IndexByEventType
```

I guess you meant `LimitForEventId.new.up` like above class definition

Reply:
`MigrationRubyCode` used twice - pushing a fix